src/components/dialog/dialog-actions/dialog-actions.ts

```diff
@@ -0,0 +1,31 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import { SbbActionGroupElement } from '../../action-group';
+
+import style from './dialog-actions.scss?lit&inline';
+
+/**
+ * Use this component to display a footer into an `sbb-dialog` with an action group.
```

Comment:
Is it possible to override the slot description? 

Otherwise in the readme.md it results as "Use the unnamed slot to add `sbb-block-link` or `sbb-button` elements to the **`sbb-action-group`**."